.flake8

```diff
@@ -2,13 +2,20 @@
 max-line-length = 120
 max-complexity = 10
 ignore =
-    E501,  # line too long, it is covered by pylint
-    E722,  # bare except, bad practice, to be removed in the future
-    F401,  # imported but unused, too many violations, to be removed in the future
-    F811,  # redefinition of unused, to be removed in the future
-    C901   # code flow is too complex, too many violations, to be removed in the future
-    W503   # line break before binary operator effect on readability is subjective
-    W504   # line break after binary operator effect on readability is subjective
+    # line too long, it is covered by pylint
+    E501,
+    # bare except, bad practice, to be removed in the future
+    E722,
+    # imported but unused, too many violations, to be removed in the future
+    F401,
+    # redefinition of unused, to be removed in the future
+    F811,
+    # code flow is too complex, too many violations, to be removed in the future
+    C901,
+    # line break before binary operator effect on readability is subjective   
+    W503,
+    # line break after binary operator effect on readability is subjective
```

Comment:
Why do you need to modify this file?

Reply:
The linter complained about the existing format (inline comments and missing commas), so it was modified to keep the linter happy. But you're right, this isn't our file to worry about, so I've reverted it to how it was.